libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -16,6 +18,8 @@ import java.net.URI
  * One of (driving|driving-traffic|walking|cycling|truck).
  * @param tilesVersion version of tiles, chosen automatically if empty
  * @param filePath used for storing road network tiles
```

Comment:
This isn't true if the `TileStore` instance is provided, can we clarify what this is used for?

Reply:
fixed